examples/measure_methods/fromFile_ADCIRCMap/BET_Q_2D.py

```diff
@@ -1,13 +1,14 @@
-# Copyright (C) 2014-2019 The BET Development Team
+# Copyright (C) 2014-2020 The BET Development Team
 
 import bet.calculateP.calculateP as calcP
+import bet.sampling.basicSampling as bsam
 import bet.calculateP.simpleFunP as sfun
 import numpy as np
 import scipy.io as sio
 import bet.sample as sample
 
 # Import "Truth"
-mdat = sio.loadmat('../matfiles/Q_2D')
+mdat = sio.loadmat('../Q_2D')
```

Comment:
```suggestion
mdat = sio.loadmat('Q_2D')
```

fixes a load error (data file now in same path as example).